hil/cli/network.py

```diff
@@ -0,0 +1,81 @@
+"""Commands related to networks are in this module"""
+import click
+import sys
+from hil.cli.client_setup import setup_http_client
+
+C = setup_http_client()
+
+
+@click.group()
+def network():
+    """Commands related to network"""
+
+
+@network.command(name='create', short_help='Create a new network')
+@click.argument('network')
+@click.argument('owner')
+@click.option('--access')
```

Comment:
Maybe here we could do something like 
```
@click.option('--access', help='Project nodes that can access this network.')

That way, `TEXT` would also have a helpful prompt.
```
Found this from a thread here https://stackoverflow.com/questions/39359420/type-and-default-input-value-of-a-click-option-in-help-option